<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setTempClass</class>
 <widget class="QDialog" name="setTempClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>setTemp</string>
  </property>
  <widget class="QLabel" name="rmInitTemplabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Temp Zone</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rmInitTemplabel_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Temp</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rmInitTemplabel_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ACTUAL TEMP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rmInitTemplabel_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Detector</string>
   </property>
  </widget>
  <widget class="QLabel" name="rmInitTemplabel_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample Preheat</string>
   </property>
  </widget>
  <widget class="QLabel" name="rmInitTemplabel_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Methanizer</string>
   </property>
  </widget>
  <widget class="QLabel" name="rmInitTemplabel_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Column</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="tempdetector">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>250.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLCDNumber" name="actualdetector">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="tempSampTee">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>250.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLCDNumber" name="actualmeth">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="tempcolumn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>300.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLCDNumber" name="actualcolumn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="tempMeth">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>450.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLCDNumber" name="actualaux">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="busyLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="settempEnterButton">
   <property name="geometry">
    <rect>
     <x>382</x>
     <y>240</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>41</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>ENTER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tempRead">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>41</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>READ</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>settempEnterButton</sender>
   <signal>released()</signal>
   <receiver>setTempClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tempRead</sender>
   <signal>released()</signal>
   <receiver>setTempClass</receiver>
   <slot>readTemp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setTempClass</sender>
   <signal>accepted()</signal>
   <receiver>tempRead</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>readTemp()</slot>
 </slots>
</ui>
